src/theme/css/chrome.css

```diff
@@ -107,7 +104,7 @@ a > .hljs {
     overflow: hidden;
     text-overflow: ellipsis;
 }
-.js .menu-title {
+.menu-title {
     cursor: pointer;
```

Comment:
If I understand it correctly, this change will result in the :point_up_2: cursor being used on the menu title, even when javascript is disabled. I don't think clicking on the title actually does anything when javascript is disabled, so you'd need to do something to prevent that.

Reply:
Good point. We could emulate the same behaviour (scrolling to the top) without JS by replacing this element with `<a href="#">...</a>`. Should I do it in this PR?

Reply:
I think that should be a separate PR.

Reply:
:+1: